crates/router/src/core/routing/helpers.rs

```diff
@@ -238,7 +170,7 @@ pub async fn update_merchant_active_algorithm_ref(
 
     Ok(())
 }
-
+// TODO: Move it to business_profile
```

Comment:
is this needed?

Reply:
Same its supposed to be handled later on